larray/core.py

```diff
@@ -6713,9 +6714,11 @@ def method(self, *args, **kwargs):
         >>> # or equivalently
         >>> # barr.all('a0,a1>>a01;a2,a3>>a23')
         """.format(_doc_agg_method("AND reduction", kwargs="out,skipna,keepaxes"))
+        pass
```

Comment:
This is very strange to use a decorator over a function and not use the decorated function at all, except its name and signature. Is there no other way?